sqlite3-to-mysql

```diff
@@ -7,6 +7,7 @@ sed \
 -e 's/"/`/g' \
 -e 's/CREATE TABLE \(`\w\+`\)/DROP TABLE IF EXISTS \1;\nCREATE TABLE \1/' \
 -e 's/\(CREATE TABLE.*\)\(PRIMARY KEY\) \(AUTOINCREMENT\)\(.*\)\();\)/\1AUTO_INCREMENT\4, PRIMARY KEY(id)\5/' \
--e "s/'t'/1/g" \
+-e "s/\b'\b/\\\'/g" \
```

Comment:
Single quotes should be ANSI SQL compliant in both dialects. Sqlite exports single quotes in a string to ```''```.